lib/ui/hooks.dart

```diff
@@ -150,7 +150,15 @@ void _updateAccessibilityFeatures(int values) {
 
 @pragma('vm:entry-point')
 void _dispatchPlatformMessage(String name, ByteData data, int responseId) {
-  if (window.onPlatformMessage != null) {
+  if (name == ChannelBuffers.kControlChannelName) {
+    try {
+      channelBuffers.handleMessage(data);
+    } catch (ex) {
+      _printDebug('Message "$name" caused exception $ex');
```

Comment:
This here would say

"Message dev.flutter/channel-buffers caused exception... " right? Maybe Message to dev.flutter/channel-buffers?

Reply:
done